### ISSUE TITLE
A question about Contact background import

### ISSUE THREAD

**Issue by hmd13** [mautic/mautic#5158]
| Q  | A
| ---| ---
| Feature request? | 
| Enhancement? | 

## Description:
Hello, I am new here, and I have a problem to understand  the automatic upload and import of Contact CSV files in Background mode.

In documentation, there's only a cron job on how to import them after being created/uploaded automatically, is there any cron job or other mean to upload these files automatically ? Thank you.

**Comment by pstevens71** [comment]
I also have this question.  I uploaded a CSV and 24 hours later, sill queued.  Do I need a cron job to do this?

**Comment by dbhurley** [comment]
Yep, check cron jobs.

**Comment by pstevens71** [comment]
Yes thank you I did get to the bottom of it.  If anyone else is wondering the cron job you need to process imports is:

/app/console mautic:import

**Comment by jmfonseca91** [comment]
@dbhurley Shouldn't this be in the documentation on Cron Jobs?

https://www.mautic.org/docs/en/setup/cron_jobs.html
